apps/web/pages/v2/apps/installed/[category].tsx

```diff
@@ -85,32 +87,42 @@ function ConnectOrDisconnectIntegrationButton(props: {
 }
 
 interface IntegrationsContainerProps {
-  variant?: "calendar" | "conferencing" | "payment";
+  variant?: "calendar" | "conferencing" | "payment" | "automation";
   exclude?: App["variant"][];
 }
 
-const IntegrationsList = ({ data }: { data: inferQueryOutput<"viewer.integrations"> }) => {
+interface IntegrationsListProps {
+  variant?: "calendar" | "conferencing" | "payment" | "automation";
+  data: inferQueryOutput<"viewer.integrations">;
+}
+
+const IntegrationsList = ({ data, variant }: IntegrationsListProps) => {
+  const seperatedItems = variant === "automation";
   return (
-    <List noBorderTreatment>
+    <List className={classNames(seperatedItems ? "flex flex-col gap-6" : "")} noBorderTreatment>
       {data.items.map((item) => (
-        <IntegrationListItem
-          name={item.name}
-          slug={item.slug}
-          key={item.title}
-          title={item.title}
-          logo={item.logo}
-          description={item.description}
-          actions={
-            <div className="flex w-32 justify-end">
-              <ConnectOrDisconnectIntegrationButton
-                credentialIds={item.credentialIds}
-                type={item.type}
-                isGlobal={item.isGlobal}
-                installed
-              />
-            </div>
-          }
-        />
+        <>
+          <IntegrationListItem
+            name={item.name}
+            slug={item.slug}
+            key={item.title}
+            title={item.title}
+            logo={item.logo}
+            description={item.description}
+            seperated={seperatedItems}
+            actions={
+              <div className="flex w-16 justify-end">
+                <ConnectOrDisconnectIntegrationButton
+                  credentialIds={item.credentialIds}
+                  type={item.type}
+                  isGlobal={item.isGlobal}
+                  installed
+                />
+              </div>
+            }>
+            {seperatedItems && <AppSettings slug={item.slug} />}
```

Comment:
It would make more sense to use variable name 
`hasAppSettings` here.

and maybe we can unconditionally use `AppSettings` component, as it is capable to take care of an app that doesn't have component defined in AppSettings Component

Reply:
@hariombalhara If we remove that, it correctly wouldn't show the AppSettings for apps that don't have the component defined but it still is counted as children of the IntegrationListItem. This means that the design for expanded IntegrationListItems is used. That was the reason why I added that. But in case we would make all apps separated in general, this would be irrelevant anyways